draft-ietf-bfcpbis-rfc4583bis.xml

```diff
@@ -163,11 +158,11 @@
       <t><list style="hanging">
           <t hangText="c-only:">The endpoint is willing to act as floor control client.</t>
           <t hangText="s-only:">The endpoint is willing to act as floor control server only.</t>
-          <t hangText="c-s:">The endpoint is willing to act as floor control client and floor control server.</t>
+          <t hangText="c-s:">The endpoint is willing to act as floor control client or floor control server.</t>
```

Comment:
This should not be in the "official" list, but instead should be text after the list, e.g.

In RFC4583, there was a third option specified, "c-s", which meant that an endpoint was willing to act as floor control client and floor control server. This feature was not implemented as specified, and the implementations which did signal "c-s" used it to mean that the endpoint is willing to act as floor control client or floor control server. An implementation of BFCPv2 MUST NOT send "c-s" as a floorctl attribute value, but MAY accept "c-s" as equivalent to "c-only s-only".

Reply:
I will fix as suggested (with the editorial changes discussed on the mailing list).